shopelectro/tests/tests_views.py

```diff
@@ -527,6 +526,26 @@ def test_page_db_template_with_special_chars(self):
         self.assertEqual(200, response.status_code)
 
 
+@tag('fast')
+class IndexPage(TestCase):
+
+    MAIN_PAGE_TILE = {
+        'some_section': [
+            {'name': 'Has url', 'url': '/section/first/'},
+            {'name': 'Has url and slug',  'slug': 'slug', 'url': '/section/second/'},
+            {'name': 'Has slug, but not url', 'slug': 'third'},
+        ]
+    }
+
+    def test_get_category_tile(self):
+        with override_settings(MAIN_PAGE_TILE=self.MAIN_PAGE_TILE):
```

Comment:
You can use it as decorator

Reply:
@artemiy312 , y, i can, but in this case i'll need to move `MAIN_PAGE_TILE` outside of class context

Reply:
@duker33 `MAIN_PAGE_TILE` and `override_settings` will be in the same scope, so there won't be an issue  
```python
def a_deco(arg):
    print(arg)
    def deco_f(f):
        return f
    return deco_f
class A:
    A_VALUE = 'it will be printed'

    @a_deco(A_VALUE)
    def a(self):
        pass
```
```
it will be printed
```

Reply:
@artemiy312 , y, it's true.
I'll fix it in further PRs